ee/codegen_integration/fixtures/simple_map_node/display_data/simple_map_node.json

```diff
@@ -25,6 +26,24 @@
           "variant": "INLINE",
           "workflow_raw_data": {
             "nodes": [
+              {
```

Comment:
This is just an order change. Probably doesn't matter, I can switch it back if we prefer.